src/plugins/intel_cpu/src/nodes/mkldnn_mvn_node.cpp

```diff
@@ -781,6 +843,71 @@ void MKLDNNMVNNode::initSupportedPrimitiveDescriptors() {
     pushDesc(LayoutType::ncsp, impl_type);
 }
 
+MKLDNNMVNNode::MVNExecutor::MVNExecutor(const MVNAttrs& mvnAttrs) :
+        shape5D(mvnAttrs.shape5D), initAcrossChannels_(mvnAttrs.initAcrossChannels_), execAcrossChannels_(mvnAttrs.execAcrossChannels_),
+        normalizeVariance_(mvnAttrs.normalizeVariance_), epsValue_(mvnAttrs.epsValue_), epsMode_(mvnAttrs.epsMode_),
+        src_data_size(mvnAttrs.src_data_size), dst_data_size(mvnAttrs.dst_data_size), is_nhwc(mvnAttrs.is_nhwc) {
+}
+
+MKLDNNMVNNode::MVNJitExecutor::MVNJitExecutor(const MVNAttrs& mvnAttrs,
+                                              const jit_mvn_config_params& jcp,
+                                              const mkldnn::primitive_attr& attr):
+                                              MVNExecutor(mvnAttrs) {
+    is_ncsp = jcp.planar_layout;
+    auto jcp_new = jcp;
```

Comment:
why we need jcp_new? I think we can use jcp directly.

Reply:
Because we need to change jcp.normalize_variance when create mvn_mean_kernel and mvn_variance kernel but key.jcp is const, so I copy jcp to jcp_new, and use jcp_new to create kernels.